client/my-sites/guided-transfer/host-credentials-page/error-notice.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import supportUrl from 'lib/url/support';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getGuidedTransferError } from 'state/sites/guided-transfer/selectors';
+
+const ErrorNotice = localize( ( { translate, errorCode } ) => {
+	if ( ! errorCode ) {
+		return null;
+	}
+
+	const errorList = {
+		'exceeded-max-validation-attempts':
+			translate( "You've entered invalid details one too many times. Please contact support." ),
+		'verify-details-failed':
+			translate( 'Those hosting details are incorrect. Please try again or contact support.' ),
+		'details-already-validated':
+			translate( "We've already confirmed your details. Please contact support if you need to change them." ),
+	};
+
+	const errorText = get(
```

Comment:
A case to be made for a `switch` with `default` here would be that you could avoid the `translate` call for the default value that you're always making here, since `translate` is a non-trivial function call. Not worried enough to warrant any change.